### ISSUE TITLE
Integrate Sanity Studio Secrets?

### ISSUE THREAD

**Issue by AllanPooley** [marco-land/sanity-plugin-vimeo-field#1]
Hey Marco,

Thanks for building this - I know more than one person that would find this enhancement super useful (myself included).

This plugin popped up on #i-made-this on the Sanity Slack Channel (that's how I found it). I could be wrong, but it looks like you're not a member there. Just thought I'd pass on a comment / suggested from [Knut Melvær](https://github.com/kmelve) (dev rel @ Sanity) 

> This is great!
Btw, I’d look into integrating https://github.com/sanity-io/sanity-studio-secrets so that folks don’t need to embed the access token in the (public) Studio JS bundle.

**Comment by marco-land** [comment]
Hi Allan,

Thanks for your feedback and for passing on Knut's comment -- I wouldn't have seen that and will also be joining the Slack community. 

Knut's comment is a good idea and I will implement it in the next release 🙂

**Comment by intelligence** [comment]
Any progress on this?

**Comment by marco-land** [comment]
Yes and no. I've put this on hold because the sanity studio secrets had [issues](https://github.com/sanity-io/sanity-studio-secrets/issues/12) which have been resolved in the meantime but I did not have time to work on this yet. I will try and add it soon though.

**Comment by intelligence** [comment]
Thanks Marco! Sorry for stressing. I've had issues to get the plugin to pickup from .env files so I'm currently putting the key right within the config, thus I was wondering :)

**Comment by marco-land** [comment]
Done with the new release https://github.com/marco-land/sanity-plugin-vimeo-field/releases/tag/v2.2.0
